### DISCUSSION TITLE
Runelite (Intel) Overlay / Screen Issue?

### DISCUSSION THREAD

**Discussion by Raw2324** [runelite/runelite#18015] [Q&A]
Hello all, 

First time using RuneLite & Github. 

I Recently upgraded my Runescape account and have decided to try the Runelite application, however when i run the application I get a graphic glitch almost like an overlay, i'm able to function in game with no problem. Does anyone have a fix for this issue? i'm able to run the regular Jagex application no problems. 

I've Attached a photo to show what my screen looks like when i open the application. 

(Running it on a Macbook Pro 2016, i7, 16gb, intel hd graphics 530)
<img width="1677" alt="Screen Shot 2024-07-06 at 4 30 44 PM" src="https://github.com/runelite/runelite/assets/174906840/01b346f6-e0b9-4c29-a6e2-b71e041a363e">

**Comment by da-loa** [comment]
Try using the 117HD or GPU plugin.  I'd mess with the settings to see if that fixes it.  Otherwise, maybe see if you have a driver update for your GPU.
